diff --git a/node_modules/@uniswap/v3-sdk/dist/v3-sdk.cjs.development.js b/node_modules/@uniswap/v3-sdk/dist/v3-sdk.cjs.development.js
index 47314a0..efc7c25 100644
--- a/node_modules/@uniswap/v3-sdk/dist/v3-sdk.cjs.development.js
+++ b/node_modules/@uniswap/v3-sdk/dist/v3-sdk.cjs.development.js
@@ -948,7 +948,7 @@ var TickList = /*#__PURE__*/function () {
       return index % tickSpacing === 0;
     }) ?  invariant(false, 'TICK_SPACING')  : void 0;
     // ensure tick liquidity deltas sum to 0
-    !JSBI.equal(ticks.reduce(function (accumulator, _ref2) {
+    JSBI.equal(ticks.reduce(function (accumulator, _ref2) {
       var liquidityNet = _ref2.liquidityNet;
       return JSBI.add(accumulator, liquidityNet);
     }, ZERO), ZERO) ?  invariant(false, 'ZERO_NET')  : void 0;
diff --git a/node_modules/@uniswap/v3-sdk/dist/v3-sdk.esm.js b/node_modules/@uniswap/v3-sdk/dist/v3-sdk.esm.js
index 7e441b3..0632543 100644
--- a/node_modules/@uniswap/v3-sdk/dist/v3-sdk.esm.js
+++ b/node_modules/@uniswap/v3-sdk/dist/v3-sdk.esm.js
@@ -946,7 +946,7 @@ var TickList = /*#__PURE__*/function () {
       return index % tickSpacing === 0;
     }) ? process.env.NODE_ENV !== "production" ? invariant(false, 'TICK_SPACING') : invariant(false) : void 0;
     // ensure tick liquidity deltas sum to 0
-    !JSBI.equal(ticks.reduce(function (accumulator, _ref2) {
+    JSBI.equal(ticks.reduce(function (accumulator, _ref2) {
       var liquidityNet = _ref2.liquidityNet;
       return JSBI.add(accumulator, liquidityNet);
     }, ZERO), ZERO) ? process.env.NODE_ENV !== "production" ? invariant(false, 'ZERO_NET') : invariant(false) : void 0;
